README.md

```diff
@@ -1,14 +1,14 @@
 # UMD
+## About
+Usermode Driver for Tenstorrent AI Accelerators
 
 ## Dependencies
 Required Ubuntu dependencies:
 ```
-sudo apt install -y libhwloc-dev
+sudo apt install -y libhwloc-dev cmake ninja-build
```

Comment:
Did you test that these are the only ones needed on a clean image?

Reply:
No - there's some compiler stuff to sort out and document.
If we don't enforce any particular compiler, then
`sudo apt install -y git build-essential cmake ninja-build libhwloc-dev`
is sufficient

Reply:
Ok we can figure it out then as part of #136